catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/Metacard.java

```diff
@@ -247,7 +247,8 @@
     /**
      * Sets {@link Attribute} with new attribute.
      *
-     * @param attribute new {@link Attribute} to set
+     * @param attribute new {@link Attribute} to set. If the getValue() call on the passed in
+     * attribute returns <code>null</code>, the attribute will be removed from this {@link Metacard}.
```

Comment:
FYI: Usually in Javadoc, code is marked as `{@code null}`. Both forms generate the same HTML.